mlir/lib/Dialect/Rock/Transforms/GemmToGridwise.cpp

```diff
@@ -93,6 +106,135 @@ struct AttentionRewritePattern : public OpConversionPattern<AttentionOp> {
                                 Value queries, Value keys, Value values) const;
 };
 
+template <typename Op>
+static LogicalResult
+computeGridSizeAttentionGemmElmtGemm(ConversionPatternRewriter &rw, Op op,
+                                     Value a, Value b, Value c) {
+  RockAccelTuningParamAttrInterface accelParams0 =
+      cast<RockAccelTuningParamAttrInterface>(op.getParams0Attr());
+
+  SmallVector<int64_t, 3> aShape =
+      llvm::to_vector<3>(cast<MemRefType>(a.getType()).getShape());
+
+  SmallVector<int64_t, 3> bShape =
+      llvm::to_vector<3>(cast<MemRefType>(b.getType()).getShape());
+
+  SmallVector<int64_t, 3> cShape =
+      llvm::to_vector<3>(cast<MemRefType>(c.getType()).getShape());
+
+  GemmSize gemm0Size(/*g=*/aShape[0], /*m=*/bShape[2],
+                     /*k=*/aShape[1],
+                     /*n=*/aShape[2]);
+  GemmSize gemm1Size(/*g=*/aShape[0], /*m=*/cShape[2],
+                     /*k=*/cShape[1],
+                     /*n=*/aShape[2]);
```

Comment:
Is it it `(a * b) * c` 
therefore `m` for the gemm1 should be coming from matrix A ?